README.md

```diff
@@ -64,6 +64,14 @@ imagor endpoint is a series of URL parts which defines the image operations, fol
 - `filters` a pipeline of image filter operations to be applied, see filters section
 - `IMAGE` is the image path or URI
   - For image URI that contains `?` character, this will interfere the URL query and should be encoded with [`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) or equivalent
+  - Base64 Option: If you prefix the IMAGE with `b64:`, you can alternatively encode the image URL as
+    [base64url](https://www.rfc-editor.org/rfc/rfc4648.html#section-5), i.e. an un-padded base64 encoding
+    with the characters `+` and `/` replaced with `-` and `_` respectively. This encoding is helpful if
+    you have lots of special characters in your image URL, and you want to avoid URL encoding them - as
+    this might lead to issues with some proxies or client libraries.
+  - For example:
+    - `b64:Z29waGVyLnBuZw` means `gopher.png`
+    - `b64:L2Zvby9nb3BoZXIgKHdpdGggc3BlY2lhbD8gY2hhcmFjdGVycykucG5n` means `/foo/gopher (with special? characters).png` 
```

Comment:
what about phased it like 

Base64 Option: Prefix your image URL with `b64:` to use [base64url]( https://developer.mozilla.org/en-US/docs/Glossary/Base64#url_and_filename_safe_base64 ) encoding instead of the raw URL. This is useful when your image URL contains special characters that might cause issues with proxies or URL encoding.

Or even simpler

Base64 URLs: Use `b64:` prefix to encode image URLs with special characters as [base64url]( https://developer.mozilla.org/en-US/docs/Glossary/Base64#url_and_filename_safe_base64 ).

What about linking to this instead https://developer.mozilla.org/en-US/docs/Glossary/Base64#url_and_filename_safe_base64 as MDN seems to explain it more clearly. Then we can omit explaining base64url. What do you think?